Exercise7-JDBC Connection/src/data/ConnectionDB.java

```diff
@@ -0,0 +1,51 @@
+package data;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Time;
+
+public class ConnectionDB {
+	  private Connection connect = null;
+      private Statement statement = null;
+      private ResultSet resultSet = null;
+     
+      public ConnectionDB() throws SQLException{
+      try {
+		Class.forName("com.mysql.jdbc.Driver");
+	    connect = DriverManager.getConnection("jdbc:mysql://localhost/high_school2","root","Globant2016");
+	    
+	    
+	  } catch (ClassNotFoundException e) {
+		e.printStackTrace();
+	  }
+      statement = connect.createStatement();
```

Comment:
I've changed it and created a new public method in the class to consult teachers